moda-contracts/src/CatalogFactory.sol

```diff
@@ -0,0 +1,50 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.21;
+
+import "./interfaces/ICatalogFactory.sol";
+import "./interfaces/ICatalog.sol";
+import "@openzeppelin/contracts/proxy/beacon/BeaconProxy.sol";
+
+contract CatalogFactory is ICatalogFactory {
```

Comment:
I don't think we need this contract. Organizations should be able to deploy their own custom Catalogs, then request membership in the ModaRegistry

Reply:
Removed.